server/services/workAndSkillsPageService.ts

```diff
@@ -15,13 +15,41 @@ import { properCaseName } from '../utils/utils'
 import { formatDate } from '../utils/dateHelpers'
 import { RestClientBuilder } from '../data'
 
+interface WorkAndSkillsData {
```

Comment:
`WorkAndSkillsData` is the data type that is returned from the public `get` function in this class (which is what the controller calls)
This definition allows us to "see" / visualise the data, as we will be making changes to this class

Reply:
Also, I've deliberately not `export`ed them as I'm not sure there's a use case (yet?) to work with these types outside of this class
Defining them in this way was more about having them defined so we have a degree of type safety, and allowing developers to discover and understand the data types used in this class.

Having said that, I'm more than happy to lift these out into the sibling `interfaces` folder and `export` them, if the connect DPS dev team would prefer it that way?